src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -467,41 +335,46 @@ const WalletTable = ({
                             ))}
                           </Flex>
 
-                          {/* Device labels */}
-                          <Flex
-                            alignItems="center"
-                            gap={3}
-                            display={deviceLabels.length > 0 ? "flex" : "none"}
-                          >
-                            <Icon as={DevicesIcon} fontSize="2xl" />
-
-                            <Text
-                              fontSize="1rem !important"
-                              fontWeight="normal !important"
+                          <Stack gap={2} mb={{ base: 0, md: 3 }}>
+                            {/* Device labels */}
+                            <Flex
+                              alignItems="center"
+                              gap={3}
+                              display={
+                                deviceLabels.length > 0 ? "flex" : "none"
+                              }
                             >
-                              {deviceLabels.join(" · ")}
-                            </Text>
-                          </Flex>
-
-                          {/* Supported languages */}
-                          <Flex alignItems="center" gap={3}>
-                            <Icon as={LanguagesIcon} fontSize="2xl" />
-
-                            <Text
-                              fontSize="1rem !important"
-                              fontWeight="normal !important"
-                            >
-                              {formatSupportedLanguages(supportedLanguages)}
-                            </Text>
-                          </Flex>
-
-                          {/* Wallet Website (desktop) */}
+                              <Icon as={DevicesIcon} fontSize="2xl" />
+
+                              <Text
+                                fontSize="1rem !important"
+                                fontWeight="normal !important"
+                              >
+                                {deviceLabels.join(" · ")}
+                              </Text>
+                            </Flex>
+
+                            {/* Supported languages */}
+                            <Flex alignItems="center" gap={3}>
+                              <Icon as={LanguagesIcon} fontSize="2xl" />
+
+                              <Text
+                                fontSize="1rem !important"
+                                fontWeight="normal !important"
+                              >
+                                {formatSupportedLanguages(supportedLanguages)}
```

Comment:
Currently we're incorrectly showing that a wallet only supports the current language, when they infact only support English.

<img width="626" alt="Screenshot 2024-02-29 at 16 00 26" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/a0c0435e-5c79-464c-ba55-363ac661b9e6">

<img width="1157" alt="Screenshot 2024-02-29 at 16 00 32" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/9fa223fb-6fa6-42c0-b3b4-9d790de4eaae">

Reply:
Fixed